src/mtx/message.ts

```diff
@@ -12,7 +12,8 @@ import getBytesFromBase64 from '../utils/getBytesFromBase64.js';
 import getHexFromBytes from '../utils/getHexFromBytes.js';
 import getBase64FromBytes from '../utils/getBase64FromBytes.js';
 import mergeUint8Arrays from '../utils/mergeUint8Arrays.js';
-import {aes, arrayStuff, arrayUnstuff, calculateCrcBytes} from './utils/crypto.js';
+import {aes, calculateCrcBytes} from './utils/crypto.js';
```

Comment:
how about use the new calculateCrc16 function?

Reply:
calculateCrcBytes implemented with calculateCrc16